server/chat-plugins/scavengers.ts

```diff
@@ -1691,6 +1691,7 @@ const ScavengerCommands: ChatCommands = {
 		}
 		if (!target && this.cmd !== 'queuerecycled') {
 			if (this.cmd === 'queue') {
+				if (!this.runBroadcast()) return false;
```

Comment:
Also, `runBroadcast()` throws if there's an problem, instead of returning, so you don't need to `return false;`.